libs/typings/src/sdk/notification.ts

```diff
@@ -0,0 +1,38 @@
+import * as PushProtocol from '@pushprotocol/restapi';
+import { z } from 'zod';
+
+export type Setting = {
+  name: string;
+  enabled: boolean;
+};
+
+export const InitializeOptionsSchema = z
+  .object({
+    readonly: z.boolean(),
+  })
+  .default({
+    readonly: true,
+  });
+
+export type InitializePushProtocolOptions = z.infer<typeof InitializeOptionsSchema>;
+
+const NotificationSettingTypeSchema = z.object({
```

Comment:
typings are ok but we should avoid keeping also the validators here(extra overhead)

Reply:
I mean in the `typings` package